Make.config

```diff
@@ -440,17 +440,17 @@ DOTNET_NUPKG_DIR ?= $(DOTNET_DESTDIR)/nupkgs
 DOTNET_PKG_DIR ?= $(DOTNET_DESTDIR)/pkgs
 DOTNET_FEED_DIR ?= $(DOTNET_DESTDIR)/nuget-feed
 
-# Configuration for .NET 5.
+# Configuration for .NET 6.
 # We're using preview versions, and there will probably be many of them, so install locally (into builds/downloads) if there's no system version to
 # avoid consuming a lot of disk space (since they're never automatically deleted). The system-dependencies.sh script will install locally as long
 # as there's a TARBALL url.
-DOTNET5_VERSION=5.0.100-rtm.20509.5
-DOTNET5_VERSION_BAND=$(firstword $(subst -, ,$(DOTNET5_VERSION)))
-DOTNET5_URL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/5.0.100-rtm.20509.5/dotnet-sdk-5.0.100-rtm.20509.5-osx-x64.pkg
-DOTNET5_TARBALL=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/5.0.100-rtm.20509.5/dotnet-sdk-5.0.100-rtm.20509.5-osx-x64.tar.gz
-DOTNET5_TARBALL_NAME=$(notdir $(DOTNET5_TARBALL))
-DOTNET5_DIR=$(abspath $(TOP)/builds/downloads/$(basename $(basename $(DOTNET5_TARBALL_NAME))))
-DOTNET5=$(DOTNET5_DIR)/dotnet
+DOTNET6_VERSION=6.0.100-alpha.1.20559.4
```

Comment:
We ended up landing with `6.0.100-alpha.1.20562.2`:

https://github.com/xamarin/xamarin-android/commit/6f761ea854f043f88dd41bb25bac608c7a680dfe

It was the first build that fixed the MSBuild bug with spaces in properties.

Reply:
Ok, trying that!